app/assets/javascripts/application.js

```diff
@@ -13,3 +13,133 @@
 //= require jquery
 //= require jquery_ujs
 //= require bootstrap
+
+/*
+  Bootstrap - File Input
```

Comment:
This should be in its own javascript file and not in `application.js` - that serves as manifest ..something like a setup file.